include/gba_video.h

```diff
@@ -38,7 +38,7 @@
 
 /*! \def BG_COLORS
 */
-#define BG_COLORS		((u16 *)0x05000000)	// Background color table
+#define BG_COLORS		((volatile u16 *)0x05000000)	// Background color table
```

Comment:
Use `vu16`. Same for others in this file

Reply:
done